src/main.js

```diff
@@ -19,9 +19,14 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
+import { generateFilePath } from '@nextcloud/router'
+
 import Vue from 'vue'
 import App from './App'
 
+// eslint-disable-next-line
+__webpack_public_path__ = generateFilePath('notestutorial', '', 'js/')
```

Comment:
```suggestion
__webpack_public_path__ = generateFilePath(appName, '', 'js/')
```

Reply:
Thanks, I did not understand that `appName` was a real accessible variable !